requirements/static/ci/common.in

```diff
@@ -2,7 +2,7 @@
 # We can also exclude platforms from the requirements using markers, but if a requirement only applies
 # to a particular platform, please add it to the corresponding `<platform>.in` file in this directory.
 azure==4.0.0; sys_platform != 'win32'
-apache-libcloud>=2.4.0; sys_platform != 'win32'
+apache-libcloud>=1.5.0; sys_platform != 'win32'
```

Comment:
Why are you downgrading the version here?

Reply:
We're only downgrading the **minimum** requirement, the actual requirement versions are the same.
@bryceml mentioned that for some distros, we still package 1.5.0, hence this change.